core/include/definitions/algebra.hpp

```diff
@@ -17,42 +17,42 @@
 namespace traccc
 {
 
-    std::array<scalar, 2> operator*(const std::array<scalar, 2> &a, scalar s)
+    inline std::array<scalar, 2> operator*(const std::array<scalar, 2> &a, scalar s)
```

Comment:
This is something we should probably harmonize with `detray`. @niermann999 @asalzburger

Reply:
Sure thing. I have found these need to be inlined to avoid link-time conflicts, but if there is a better solution that is fine by me.